### PR TITLE
Check if hostname is available when incrementing it

### PR DESCRIPTION
**Pull request by romcheg**

### CODE REVIEW

File: src/ralph/networks/tests/test_models.py [hunk 1]
```diff
@@ -363,6 +387,31 @@ def test_get_next_hostname(self):
         self.assertEqual(ne.next_free_hostname, 's12300001.dc.local')
         self.assertEqual(ne.next_free_hostname, 's12300001.dc.local')
 
+    def test_get_next_hostname_conciders_other_models(self):
```
Comment (ghost) [inline src/ralph/networks/tests/test_models.py]: `conciders`?

Comment (romcheg) [reply src/ralph/networks/tests/test_models.py]: That's one of the words I will probably never be able to spell correctly

File: src/ralph/networks/tests/test_models.py [hunk 2]
```diff
@@ -339,6 +341,28 @@ def test_issue_next_hostname(self):
         self.assertEqual(ne.issue_next_free_hostname(), 's12300005.dc.local')
         self.assertEqual(ne3.issue_next_free_hostname(), 's100001.dc.local')
 
+    def test_issue_next_hostname_hostname_taken(self):
```
Comment (ghost) [inline src/ralph/networks/tests/test_models.py]: `test_issue_next_hostname_hostname_taken`? :)

Comment (romcheg) [reply src/ralph/networks/tests/test_models.py]: @xliiv would test_issue_next_hostname_when_a_few_next_hostnames_are_taken be better?

Comment (ghost) [reply src/ralph/networks/tests/test_models.py]: yes

File: src/ralph/networks/tests/test_models.py (hunk 1 above)
Comment (ghost) [reply src/ralph/networks/tests/test_models.py]: yea, english spelling is shit..

**Review by mkurek — APPROVED** [review]

**Review by ghost — APPROVED** [review]

**Pull request merged**